vsts/node-release.yaml

```diff
@@ -0,0 +1,185 @@
+name: Release_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+
+phases:
+
+- phase: Phase_1
+  displayName: Create Release Branch
+
+  condition: succeededOrFailed()
+  queue:
+    name: Hosted
+
+  steps:
+
+  - checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+    persistCredentials: true  # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+
+  - task: NodeTool@0
+    displayName: 'Use Node 6.x'
+    enabled: false
+
+  - task: Npm@1
+    inputs:
+      command: 'install'
+      workingDir: $(Build.SourcesDirectory)\.scripts\version
+    enabled: true
+
+  - powershell:  |
+      $configFile = "$(Build.SourcesDirectory)\.scripts\version\config.json"
+      $app = "$(Build.SourcesDirectory)\.scripts\version\app.js"
+
+      if ($(Test-Path -Path $configFile -PathType Leaf) -ne $true) {
+          throw "File not found: $($configFile)"
+      }
+
+      if ($(Test-Path -Path $app -PathType Leaf) -ne $true) {
+          throw "File not found: $($app)"
+      }
+
+      node $app $configFile "$(Build.SourcesDirectory)"
+      
+    displayName: Bump Version
+    enabled: true
+
+  - script: dir /b /s
+
+  - powershell: |
+      $releaseTag = Get-Date -UFormat "%Y-%m-%d"
+      $releaseBranch = "release_$($releaseTag)"
+
+      Set-Location "$(Agent.BuildDirectory)\s"
+
+      git config user.email $(GitHubEmail)
+      git config user.name $(GitHubName) 
+      git checkout -b $releaseBranch
+      git add .
+      git commit -m "Bump package versions for release $($releaseTag)"
+      git tag -d $($releaseTag)
+      git tag -a $($releaseTag) -m "SDK release on $($releaseTag)"
+      git push --tags -u origin $($releaseBranch)
+      git checkout master
```

Comment:
>git checkout master [](start = 6, length = 19)

careful if you checkout master now, all the files that have been modified and pushed to the release branch are now back to their "non-bumped" state. which will lead you to generate packages for the old version instead of the new one.